tokenlist.json

```diff
@@ -273,10 +218,79 @@
       "name": "Wrapped UST Token",
       "logoURI": "https://etherscan.io/token/images/terra-ust_32.png",
       "decimals": 18,
-      "tags": [
-        "wrapped",
-        "ust"
-      ]
+      "tags": ["wrapped", "ust"]
+    },
+    {
+      "chainId": 1,
+      "address": "0x09fE5f0236F0Ea5D930197DCE254d77B04128075",
+      "symbol": "WCK",
+      "name": "Wrapped CryptoKitties",
+      "logoURI": "https://assets.coingecko.com/coins/images/8797/small/WCK.png",
```

Comment:
It would be awesome if we could upload logos to IPFS and link to them, hereby not depending on web2 infrastrure.

Reply:
https://github.com/tokensoft/wrapped-tokenlist/issues/9